src/replicache.test.ts

```diff
@@ -14,6 +15,11 @@ const httpInvoke = httpInvoker.invoke.bind(httpInvoker);
 
 const invoke: RepmInvoke = httpInvoke;
 
+type TestMode = 'live' | 'replay' | 'record';
+
+// eslint-disable-next-line prefer-const
+let testMode: TestMode = 'live';
```

Comment:
Can this be `const`?

Reply:
This will read the env in an upcoming patch so that is why I made it let